apps/mymedicare_cb/views.py

```diff
@@ -93,7 +95,7 @@ def authenticate(request):
     user_info = response.json()
 
     # Set identity values from userinfo response.
-    sls_subject = user_info.get("sub", None).strip()
+    sls_subject = user_info.get("sub", "").strip()
```

Comment:
NOTE:  Was getting an exception here with the strip() function when None. The empty string is good here and will produce an exception down the line.